src/movies.js

```diff
@@ -1,22 +1,61 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
-
+function getAllDirectors(moviesArray) {
+    let film=moviesArray.map(function(cinema){
```

Comment:
Yesterday we learned a bit more about the arrow function. Yours here is correct, and if you wanted to use it, it would look like this:

```
function getAllDirectors(moviesArray) {
let film = moviesArray.map(cinema => cinema.director);
return film;
}
```
Regarding behaviour, both the function and the arrow function will have the same behaviour, but the arrow function is shorter and easier to read.